platform/innovium/sonic-platform-modules-wistron/6512-32r/sonic_platform/sfp.py

```diff
@@ -10,8 +10,14 @@
     from sonic_platform_base.sonic_sfp.sff8436 import sff8436Dom
     from sonic_platform_base.sonic_sfp.qsfp_dd import qsfp_dd_InterfaceId
     from sonic_platform_base.sonic_sfp.qsfp_dd import qsfp_dd_Dom
+    from sonic_platform_base.sonic_sfp.sffbase import sffbase
+    from sonic_platform_base.sonic_sfp.sff8024 import type_abbrv_name
+    from sonic_platform_base.sonic_sfp.sff8024 import type_of_media_interface
```

Comment:
@RogerX87 please use sonic_xcvr instead of sonic_sfp. sonic_sfp is deprecated. 
SFP-refactor HLD- https://github.com/sonic-net/sonic-buildimage/pull/9142/
Sample PR to migrate to SFP-refactor:- https://github.com/sonic-net/sonic-buildimage/pull/9142/